web/vtadmin/src/index.css

```diff
@@ -232,3 +230,121 @@ table tbody td[rowSpan] {
 .Toastify__toast-theme--dark {
     background: none !important;
 }
+
+/* See https://tailwindcss.com/docs/extracting-components */
+@layer components {
```

Comment:
I pulled this CSS from src/style/components.css, and then deleted that file.

This is because tailwindcss v3 tweaked its base button styles in a way that conflicted with our `.btn` component, as described in https://github.com/tailwindlabs/tailwindcss/issues/6602. Trying to fix it resulted in a real nasty snarl of [import ordering issues](https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports), so I've opted to simply pull everything into this one file for now.